tests/conftest.py

```diff
@@ -11,17 +9,62 @@ def runner(request):
     return CliRunner()
 
 
-def _check_symlinks_supported():
-    with tempfile.TemporaryDirectory(prefix="click-pytest-") as tempdir:
-        target = os.path.join(tempdir, "target")
-        open(target, "w").close()
-        link = os.path.join(tempdir, "link")
+@pytest.fixture(name="assert_no_surrogates", scope="session")
+def assert_no_surrogates_fixture():
```

Comment:
This doesn't appear to need to be a fixture, why isn't it in `helpers` like the other two functions?

Reply:
It allows the self-test that `UnicodeEncodeError.reason == "surrogates not allowed"` (I don't know if error messages are "stable"). I could remove that and make it importable if you prefer.

Reply:
Yes, remove this.

Reply:
changed it to a helper that's imported